### DISCUSSION TITLE
Posts in multiple paths versus single path

### DISCUSSION THREAD

**Discussion by CharliePoole** [statiqdev/Discussions#25] [Q&A]
(Apparently, I'm suddenly full of questions. I think it's because I understand enough now to ask them.)

Anyway... I have implemented something called "Topics" which is a very dumb version of what "Categories" are in other platforms. Every post belongs to one Topic. Currently, the list of available topics is hard-coded and related to the input sub-directory in which the post is found. (BTW, this grew out of the fact that I'm retired and combining my technical blog with my personal site.)

When I implemented it, my choices were limited because I didn't yet know how to set the destination for a post. Now I do. So I'm wondering if I should go back to a single source directory and use a Topic document setting to determine the output destination. I can see how to implement that pretty easily. It would make it possible for me to add and remove topics easily. I'm not concerned about breaking links yet as the site is new.

I'd love to hear comments on the pros and cons of single source versus multiple source posts.

**Comment by daveaglick** [comment]
I don't know that there's any technical reason why one approach would be better than another here, probably more which way helps you find posts easier and organize them.

Personally, if there's always going to be a one-to-one relationship between posts and topics I'd keep them the way you have them. It more closely matches how things appear in output (which is always good IMO) and helps break up a potentially long list of files.

**Comment by CharliePoole** [comment]
That was my thinking originally and so far it has worked. The test will come when I add a new topic, but that's infrequent. Thanks for thinking about it. :smile:
